sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2_1_preview_2/models/_models.py

```diff
@@ -93,6 +93,31 @@ def __init__(
         self.errors = kwargs.get('errors', None)
 
 
+class Appearance(msrest.serialization.Model):
+    """An object representing the appearance of the text line.
+
+    All required parameters must be populated in order to send to Azure.
```

Comment:
I see this is in a generated class. will our users see this comment? not sure what it is referring too as this is an output model. no?

Reply:
Unfortunately yes until we wrap this model

Reply:
oh well. ok for now